tests/rados/test_scrub_parameters.py

```diff
@@ -0,0 +1,252 @@
+    pool_type = ["replicated_pool", "ec_pool"]
+    try:
+
+        flag = wait_for_clean_pg_sets(rados_object)
+        if not flag:
+            log.error(
+                "The cluster did not reach active + Clean state after add capacity"
+            )
+            return 1
+        for val in ("scrub", "deep-scrub"):
+            for pool in pool_type:
+                pool_detail = config[pool]
+                pool_name = pool_detail["pool_name"]
+                selected_osd = create_pool_get_osd(rados_object, pool, pool_detail)
+                dump_scrub_before_scrub = scrub_object.get_dump_scrubs(selected_osd)
+                if val == "scrub":
+                    rados_object.run_scrub(osd=selected_osd)
+                else:
+                    rados_object.run_deep_scrub(osd=selected_osd)
+                status, pg_id = chk_forced_flag(scrub_object, selected_osd)
+                if not status:
+                    log.info(
+                        f"The {val}bing not started so as workaround performing the {val}bing on pg"
+                    )
+                    if val == "scrub":
+                        rados_object.run_scrub(pgid=pg_id)
+                    else:
+                        rados_object.run_deep_scrub(pgid=pg_id)
+                status, pg_id = chk_forced_flag(scrub_object, selected_osd)
+                if not status:
+                    log.error(f"After the {val} none of the pgid is set as true")
+                    return 1
+                forced_flag = wait_pg_forced_flag(scrub_object, selected_osd, pg_id)
+                if not forced_flag:
+                    log.error(
+                        "The forced flag is not set to True and the waiting time is 20 minutes"
+                    )
+                    return 1
+                log.info(f"===Scenario-1: Completed by {val}bing the osd===")
+                pg_dump_scrub = scrub_object.get_pg_dump_scrub(selected_osd, pg_id)
+                log.info(f"The {pg_id} dumb scrub is - {pg_dump_scrub}")
+                pg_current_sched_time = pg_dump_scrub["sched_time"]
+                # Get the dump_scrub of the PG id
+                scheduled_time = verify_scheduled_time(
+                    mon_object=mon_obj,
+                    prev_dump_scrub=dump_scrub_before_scrub,
+                    pg_id=pg_id,
+                    pg_current_sched_time=pg_current_sched_time,
+                )
+                if not scheduled_time:
+                    log.error("The scrub time is greater than upper bound time")
+                    return 1
+                log.info(f"===Scenario-2:Completed by {val}bing the osd")
+                log.info(f"=====Verification completed by {val}bing the OSD====")
+                # deleted pool.This is due to execution time is taking more time until waiting for the cluster to clean
+                # state
+                method_should_succeed(rados_object.delete_pool, pool_name)
```

Comment:
move the module to delete pools in finally block.

Reply:
This method already exists in the finally block.